sync_radar_data_globus.py

```diff
@@ -64,10 +49,18 @@
 # for automatic authentication
 TRANSFER_RT_FILENAME = HOME + "/.globus_transfer_rt"
 
-# UUID of your endpoint, retrieve from endpoint information at: https://www.globus.org/app/endpoints
-PERSONAL_UUID = ''
-# Client ID retrieved from https://auth.globus.org/v2/web/developers
-CLIENT_ID = ''
+# UUID of your endpoint, retrieve from endpoint info at: https://www.globus.org/app/endpoints
+# Or from the filesystem that globusconnectpersonal is installed on. Note that the following
+# line is for LINUX operating systems and may be different for MAC OS.
+# Note: this assumes you are running globusconnectpersonal & this script on the same filesystem
+PERSONAL_UUID_FILENAME = HOME + "/.globusonline/lta/client-id.txt"
+
+if isfile(PERSONAL_UUID_FILENAME):
+    with open(PERSONAL_UUID_FILENAME) as f:
+        PERSONAL_UUID = f.readline()
+
+# Client ID retrieved from https://auth.globus.oorg/v2/web/developers
+CLIENT_ID = '84d0b918-f49a-4136-a115-4206dafeba8a'
 
 
 def month_year_iterator(start_month, start_year, end_month, end_year):
```

Comment:
lines 75-76 : Maybe a comment to explain what you are calculating, day of year?

Reply:
I removed this function since it's no longer used.